static_report/src/components/ComparisonReport/CRTableList/index.tsx

```diff
@@ -43,7 +55,7 @@ export function CRTableList({
 }) {
   const { base, input: target } = data;
   const tables = transformAsNestedBaseTargetRecord<
-    SingleReportSchema['tables'],
+    SaferSRSchema['tables'],
     TableSchema
```

Comment:
`SaferTableSchema`?

Reply:
Here is a table, I think using `SafterSRSchema` is right?

Reply:
Yup. I'm talking about the one after :) 
Line 59 `TableSchema` -> `SaferTableSchema`

Reply:
LOL, I saw the highlight and misunderstood, let me fix it.